src/webapp/reports/validate-custom-forms/ValidateCustomFormsReport.tsx

```diff
@@ -0,0 +1,99 @@
+import { Typography, makeStyles } from "@material-ui/core";
```

Comment:
I forgot, I see 2 errors in the console, can you check?

```
index.js:1 Warning: Each child in a list should have a unique "key" prop.

Check the render method of `ValidateCustomFormsReport`
```

```index.js:1 Warning: validateDOMNesting(...): <div> cannot appear as a descendant of <p>.```

Reply:
I can't solve the first one too:
index.js:1 Warning: Each child in a list should have a unique "key" prop.

I need change the way to draw the error list in validateCustomFormsReports.tsx:
            {_.map(errors, item => {
                return <div className={classes.items}>{item.text}</div>;
            })}

Reply:
slved